app/views/content/train-to-be-a-teacher/who-do-you-want-to-teach.md

```diff
@@ -3,114 +3,85 @@ title: Who do you want to teach
+If you’re doing initial teacher training (ITT), you could consider options for primary teaching with a subject specialism like maths or English. This will cover all of the subjects but with a focus on your subject specialisation.
 
-### Teach pupils with sensory impairments
+### Secondary (ages 11 to 18)
+Children start secondary school when they’re 11. There are up to 3 stages in secondary school: 
 
-You need [specific qualifications to teach a class of pupils with hearing impairments, vision impairments or multi-sensory impairments](https://www.gov.uk/guidance/mandatory-qualifications-specialist-teachers).
+* key stage 3 (KS3) includes years 7 to 9 with children age 11 to 14  
+* key stage 4 (KS4) includes years 10 and 11 with pupils age 14 to 16
+* key stage 5 (KS5) includes years 12 and 13 from age 16 to 18 at secondary schools with sixth forms or colleges
 
-### Become a special educational needs coordinator
+Pupils work towards national qualifications, usually GCSEs during KS4. Students take A-levels or other similar qualifications during KS5. 
 
-A special educational needs coordinator (SENCO) assesses, plans and monitors the progress of disabled pupils and pupils with special educational needs.
+Subjects at this level include English, maths and science, computing, physical education (PE) and modern foreign languages. Schools will offer a range of other subjects which could include drama, dance, and media studies. Schools are also required to offer RE, careers guidance and sex and relationship education (SRE).
 
-Once you’re a qualified teacher you’ll need to complete the National Award in Special Educational Needs Coordination (NASENCo) when you take up your SENCO post.
+If you’d prefer to specialise in a subject but still enjoy teaching a variety of things, being a secondary school teacher might suit you.
 
-## Early years teacher training
+### What you need to teach in primary and secondary schools
 
-The early years are critical in children’s development. Your input will have a lasting and positive effect on their development and wellbeing. You’ll use your knowledge and skills to help young children enjoy high standards of teaching and open their minds to new ideas every day.
+To teach in the majority of primary and secondary schools, you will need to achieve qualified teacher status (QTS) and there are a number of routes to this. Check [what training you need to get QTS](/train-to-be-a-teacher).
 
-You’ll need to do early years initial teacher training (EYITT) to achieve early years teacher status (EYTS).
+Once you have QTS, you're qualified to teach any of these age ranges. If you want to change age groups you'll need to be able to show you have relevant classroom experience, for example being able to cover the broader curriculum if you want to switch from secondary to primary.
```

Comment:
'**these** age ranges' feels a bit non-specific - which age ranges is this referring to? are we talking about any age ranges in primary and secondary?